installers/README.txt

```diff
@@ -0,0 +1,3 @@
+To use Nu plugins, use the register command to tell Nu where to find the plugin. For example:
+
```

Comment:
big NO. Do not start writing installers in this PR. It's for CI, leave these changes for an another one, please